controllers/velero_test.go

```diff
@@ -3651,12 +3651,9 @@ func TestDPAReconciler_getVeleroImage(t *testing.T) {
 				},
 			},
 			pluginName: common.Velero,
-			wantImage:  "quay.io/konveyor/velero:latest",
+			wantImage:  "notquay.io/someorg/velero:sometag",
 			setEnvVars: map[string]string{
-				"REGISTRY":    "quay.io",
-				"PROJECT":     "konveyor",
-				"VELERO_REPO": common.Velero,
-				"VELERO_TAG":  "latest",
+				"RELATED_IMAGE_VELERO": "notquay.io/someorg/velero:sometag",
```

Comment:
@kaovilai , deliberately set invalid image?

Reply:
This is a unit test. It only checks if image returned by the function is one that is not default.

If we check the same image as hardcoded image, we wouldn't know if ENV actually work or if it came from a hardcoded backup.

Reply:
Good enough for me :sunglasses:

Reply:
UnsupportedOverrides comes first then ENV then hardcoded backup image.

You could make a release that only require a tag change and only change this env in CSV instead of having to recompile everything.

Reply:
this "invalid" image actually won't get pulled :)

Reply:
Yes, end result is the same.  I realized it was in test (and probably intentionally left as that value), but I wanted to double-check.  Thanks!